### PR TITLE
Fix procedure about configuring LMS #6273

### LINKED ISSUE #6273: Fix procedure about configuring a learning management system
### What article on docs.github.com is affected?

The “Creating a roster for your classroom” section of “[Manage classrooms](https://docs.github.com/en/education/manage-coursework-with-github-classroom/manage-classrooms#creating-a-roster-for-your-classroom)”

### What part(s) of the article would you like to see updated?

Update step 5 to clarify that there are two options here and show the time-saving option first.
```
5. Choose whether to import a roster or create a roster manually.
  - To upload a file containing the identifiers, click **Upload a CSV or text file**. 
  - Otherwise, to create a roster manually, type your student identifiers.
```
### Additional information

This issue is for a special project and is not currently available for contributors to work on.

### PR DESCRIPTION
**Pull request by AlexandraMMR**
Updated step 5 in “Creating a roster for your classroom” section of “Manage classrooms” to clarify that there are two options here and show the time-saving option first.

### Why:
To clarify that there are two options in “Creating a roster for your classroom” section of “Manage classrooms” and show the time-saving option first.

Closes [6273](https://github.com/github/docs/issues/6273)

### What's being changed:
Step 5 is now:
1. Choose whether to import a roster or create a roster manually.
  - To import a roster, upload a file containing the identifiers by clicking **Upload a CSV or text file**. 
  - Otherwise, to create a roster manually, type your student identifiers.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by AlexandraMMR** [comment]
Build looks good, fixed the spacing to match the other text so ready for review tomorrow @emilyistoofunky 👍

**Comment by janiceilene** [comment]
🎉 Thanks for opening this PR @AlexandraMMR 🎉 I've triaged it for writer review!

**Comment by Daytonshaver140** [comment]
Read is on format to permit further reference change.

File: content/education/manage-coursework-with-github-classroom/manage-classrooms.md [hunk 1]
```diff
@@ -58,8 +58,10 @@ If your course already has a roster, you can update the students on the roster o
 {% data reusables.classroom.click-students %}
 1. To connect {% data variables.product.prodname_classroom %} to your LMS and import a roster, click {% octicon "mortar-board" aria-label="The mortar board icon" %} **Import from a learning management system** and follow the instructions. For more information, see "[Connect a learning management system to {% data variables.product.prodname_classroom %}](/education/manage-coursework-with-github-classroom/connect-a-learning-management-system-to-github-classroom)."
     !["Import from a learning management system" button](/assets/images/help/classroom/click-import-from-a-learning-management-system-button.png)
-1. To create a roster manually, type your student identifiers. Optionally, click **Upload a CSV or text file** to upload a file containing the identifiers.
-    ![Text field for typing student identifiers and "Upload a CSV or text file" button](/assets/images/help/classroom/type-or-upload-student-identifiers.png)
+1. Choose whether to import a roster or create a roster manually.
```
Comment (lecoursen) [inline content/education/manage-coursework-with-github-classroom/manage-classrooms.md:61]: For a step like this, with two-sub steps, we want the original step to be a summary of all sub-steps. Choosing whether you want to import a roster or creating one manually is really a different step than doing the actual importing. However, "create your roster" is a good summary, and the user can choose to accomplish the step either by importing or typing. 😊 

```suggestion
1. Create your class roster.
```

File: content/education/manage-coursework-with-github-classroom/manage-classrooms.md [hunk 2]
```diff
@@ -58,8 +58,10 @@ If your course already has a roster, you can update the students on the roster o
...
+    - To import a roster, upload a file containing the identifiers, click **Upload a CSV or text file**.
```
Comment (lecoursen) [inline content/education/manage-coursework-with-github-classroom/manage-classrooms.md:62]:
```suggestion
    - To import a roster by uploading a file containing student identifiers, click **Upload a CSV or text file**.
```

File: content/education/manage-coursework-with-github-classroom/manage-classrooms.md [hunk 3]
```diff
@@ -58,8 +58,10 @@ If your course already has a roster, you can update the students on the roster o
...
+    - Otherwise, to create a roster manually, type your student identifiers.
```
Comment (lecoursen) [inline content/education/manage-coursework-with-github-classroom/manage-classrooms.md:63]:
```suggestion
    - To create a roster manually, type your student identifiers.
```

**Review by lecoursen — COMMENTED** [review]
Thanks for the help @AlexandraMMR! I just left a few small suggestions, which I'll apply now and then merge! ⚡

File: content/education/manage-coursework-with-github-classroom/manage-classrooms.md [hunk 4]
```diff
@@ -58,8 +58,10 @@ If your course already has a roster, you can update the students on the roster o
 {% data reusables.classroom.click-students %}
 1. To connect {% data variables.product.prodname_classroom %} to your LMS and import a roster, click {% octicon "mortar-board" aria-label="The mortar board icon" %} **Import from a learning management system** and follow the instructions. For more information, see "[Connect a learning management system to {% data variables.product.prodname_classroom %}](/education/manage-coursework-with-github-classroom/connect-a-learning-management-system-to-github-classroom)."
     !["Import from a learning management system" button](/assets/images/help/classroom/click-import-from-a-learning-management-system-button.png)
-1. To create a roster manually, type your student identifiers. Optionally, click **Upload a CSV or text file** to upload a file containing the identifiers.
-    ![Text field for typing student identifiers and "Upload a CSV or text file" button](/assets/images/help/classroom/type-or-upload-student-identifiers.png)
+1. Create your class roster.
```
Comment (lecoursen) [inline content/education/manage-coursework-with-github-classroom/manage-classrooms.md:61]:
```suggestion
1. Provide the student identifiers for your roster.
```

**Review by lecoursen — APPROVED** [review]

**Comment by AlexandraMMR** [comment]
> Thanks for the help @AlexandraMMR! I just left a few small suggestions, which I'll apply now and then merge! ⚡

Thank you for the edits Laura ✨

**Pull request merged**